service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <service verb="validate" noun="PostalAddress">
+        <in-parameters>
+            <parameter name="shippingGatewayConfigId" required="true"/>
+            <parameter name="contactMechId" required="true"/>
+            <parameter name="partyId"/>
+            <parameter name="facilityId"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="contactMechId"/>
+            <parameter name="postalAddress"/>
+        </out-parameters>
+        <actions>
+            <!-- Searching FedexApiToken -->
+            <entity-find-one entity-name="ShippingGatewayOption" value-field="apiTokenOpt">
+                <field-map field-name="shippingGatewayConfigId" from="shippingGatewayConfigId"/>
+                <field-map field-name="optionEnumId" value="FedexApiToken"/>
+            </entity-find-one>
+            <if condition="!apiTokenOpt.optionValue">
+                <return type="warning" message=" ${shippingGatewayConfigId} has no FedexApiToken, not validating address"/>
+            </if>
+
+            <!-- Converting Address Data to JSON Request Map -->
+            <service-call name="mantle.fedex.FedexServices.get#AddressMap" in-map='[contactMechId:contactMechId, partyId:partyId]'
+                          out-map='addressMap'/>
+            <if condition="!addressMap"><return/></if>
+
+            <!-- Setting Connection With Fedex -->
+            <service-call name="mantle.fedex.FedexServices.set#Connection"
+                          in-map="[method:'POST', connectUrl:'https://apis-sandbox.fedex.com/address/v1/addresses/resolve',
```

Comment:
You should not hard code the connection url, please check to store the fedex connection details.